woocommerce/woocommerce
# Fix broken Code Coverage tests

### All Submissions:






### Changes proposed in this Pull Request:
This PR updates the code coverage shell script to call PHPUnit from the correct location. It uses `$COMPOSER_BIN_DIR/phpunit` now instead of manually defining the composer install directory based on $HOME.

### How to test the changes in this Pull Request:

1. Let build complete
2. View travis logs and open the code coverage one
3. Check the logs and ensure that the code coverage actually completed and not just returning a 0.

### Other information:
